src/main/java/core/basesyntax/shapes/Square.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.shapes;
+
+public class Square extends Shape implements AreaCalculable, Drawable {
+    private double side;
+
+    public Square(String name, String color, double side) {
+        super(name, color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    @Override
+    public void draw() {
+        // Figure: square, area: 12.5 sq. units, side length: 7.1 units, color: yellow
```

Comment:
please remove commented code from all your classes where you have it

Reply:
done